lighthouse-core/report/scripts/lighthouse-report.js

```diff
@@ -47,9 +49,19 @@ LighthouseReport.prototype = {
     }
   },
 
+  toggleHelpText: function(e) {
+    if (e.target.classList.contains('report-section__item-help-toggle')) {
+      const el = e.currentTarget.parentNode.querySelector('.report-section__item-helptext');
```

Comment:
Moved the handlers up a level to each item in the report (`.report-section__item`).  That should "guarantee" the help text falls within the `currentTarget`